### ISSUE TITLE
[FEATURE REQUEST] Jobs number ?

### ISSUE THREAD

**Issue by mikafouenski** [sindresorhus/pure#334]
Hello,

It would be nice to have the number of background jobs displayed.

This prompt is realy great.

**Comment by mafredri** [comment]
I don't think this is something we want to include in Pure. IMO it would add visual noise. Then again, I don't often leave background jobs running. So perhaps it wouldn't be an issue.

Either way, for now, here's a workaround you can use: https://github.com/cosminlupu/pure/commit/4853d440dfe601d180955f4ec8af582d7f0af00e#commitcomment-20540755
